coralproject/talk
# [CORL-3022]: add new outlook accounts to protected email domains

## What does this PR do?

Adds more Outlook domains to the protected email domains list

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

* rebuild common (because the list is shared in the common library)
* rebuild server and client and spin up Coral
* attempt to spam ban a domain using:
  - outlook.co.nz
  - outlook.at
  - outlook.in
  - outlook.com.au
  - outlook.fr
  - outlook.de
  - outlook.jp

* see that each is blocked like all the other protected domains

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
